sdks/java/extensions/sql/zetasql/src/main/java/org/apache/beam/sdk/extensions/sql/zetasql/SqlAnalyzer.java

```diff
@@ -206,7 +206,37 @@ private void addBuiltinFunctionsToCatalog(SimpleCatalog catalog, AnalyzerOptions
     // TUMBLE
     catalog.addTableValuedFunction(
         new TableValuedFunction.ForwardInputSchemaToOutputSchemaWithAppendedColumnTVF(
-            ImmutableList.of("TUMBLE"),
+            ImmutableList.of(TVFStreamingUtils.FIXED_WINDOW_TVF),
+            new FunctionSignature(
+                retType, ImmutableList.of(inputTableType, descriptorType, stringType), 123),
```

Comment:
The constant `123` here (and repeated below) is suppose to be a globally unique ID out of `com.google.zetasql.ZetaSQLFunction.FunctionSignatureId`. This one is `FN_BITWISE_NOT_UINT64`. I'm pretty sure that doesn't match this function signature. If we don't care about these, could you use `com.google.zetasql.ZetaSQLFunction.FunctionSignatureId.FN_INVALID_FUNCTION_ID`.

Reply:
Good point. I changed it back to `-1`. I have seen `-1`usage in internal codebase.

Reply:
`-1` is `__FunctionSignatureId__switch_must_have_a_default__`, but I don't see any harm in using that.